### DISCUSSION TITLE
Prioritizing MoltenVK feature developments via targeted contributions and funding

### DISCUSSION THREAD

**Discussion by billhollings** [KhronosGroup/MoltenVK#2397] [General]
MoltenVK is incredibly successful, is a cornerstone of the Vulkan eco-system, and is used by many development organizations to produce games, emulators, and significant non-gaming applications using  Vulkan on Apple platforms.

We often receive requests to add significant new features.

MoltenVK is an open-source project, and as-such, is dependent on code and financial contributions from the Vulkan developer and user community.

If you or your organization are looking for features to be added to MoltenVK, and are in a position to contribute code, or funding towards those features, please reach out to us.

Two ways to do that are:

1. Respond here to this discussion post.
2. Email info@brenwill.com to initiate a direct and private discussion.

**Comment by rcaridade145** [comment]
Something that was incredibly successful with game emulators like Cemu -https://www.reddit.com/r/cemu/comments/63m4mj/team_cemu_is_now_the_3rd_best_patreon_in_the/ - was Patreon. The Zelda series when launched on the WiiU made a lot of people contribute. Wonder if a similar approach can be taken with MoltenVK .  Doom is VK only, the latest Indiana Jones is Vulkan only.
